project/index.html

```diff
@@ -0,0 +1,216 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+	<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+	<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1.0" />
+	<title>Home2Health</title>
+	<!-- CSS  -->
+	<script src="https://code.jquery.com/jquery-2.1.1.min.js"></script>
+	<script src="/js/materialize.js"></script>
+	<link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+	<link href="/css/materialize.css" type="text/css" rel="stylesheet" media="all" />
+	<link href="/css/style.css" type="text/css" rel="stylesheet" media="all" />
+</head>
+
+<body>
+	<nav class="lighten-1 white" id="header">
+		<div class="nav-wrapper container">	<a id="logo-container" href="#" class="brand-logo">Home2Health</a>
+			<ul class="right hide-on-med-and-down">
+				<li>	<a class='dropdown-trigger' href='#' data-target='dropdown2'>What's happening in Australia?
+						<i class="material-icons right">arrow_drop_down</i>
+						</a>
+					<ul id="dropdown2" class="dropdown-content">
```

Comment:
Use consistent formatting its 4 spaces here. Make sure you have prettier and make it format on save

Reply:
I deleted /project before, but merging `master` hasn't applied the changes... deleted again anyways.